views/partials/foot.jade

```diff
@@ -1,7 +1,10 @@
 if scripts && scripts.length
 	each script in scripts
 		if !script.inHead
-			script(async=script.async crossorigin=script.crossorigin defer=script.defer integrity=script.integrity src=script.src text=script.text type=script.type)
+			if script.content
+					script(async=script.async crossorigin=script.crossorigin defer=script.defer integrity=script.integrity src=script.src text=script.text type=script.type)!= script.content
+				else
```

Comment:
I think this block is indented too much?

Reply:
It just looks large here. It's using tab indentation. Looks fine in the editor, and it is inline with the script tag on L7. Looks fine to me...